src/argilla/client/feedback.py

```diff
@@ -412,7 +402,7 @@ def fetch_records(self) -> None:
             first_batch = datasets_api_v1.get_records(
                 client=httpx_client, id=self.argilla_id, offset=0, limit=FETCHING_BATCH_SIZE
             ).parsed
-            self.__records = first_batch.items
+            self.__records = [FeedbackRecord(**item.dict()) for item in first_batch.items]
```

Comment:
Should we use `FeedbackRecord.parse_obj` instead? Not sure what's the difference

Reply:
I realize that `item` is an object. Should we use [`from_orm`](https://docs.pydantic.dev/latest/usage/models/#orm-mode-aka-arbitrary-class-instances)?  So, we don't need to convert them to dict.

Reply:
Sure we can also do that, but we need to update the `Config` then, right?